packages/gasket-nextjs/package.json

```
@@ -64,15 +64,15 @@
     "jsdom": "^20.0.0",
     "mocha": "^10.0.0",
     "mock-require": "^3.0.3",
-    "next": "^12.0.8",
+    "next": "^13.1.1",
```

This will need to be pinned down for now
```suggestion
    "next": "13.1.1",
```